pkg/subsystem/lists/linux.go

```diff
@@ -253,7 +254,7 @@ func init() {
+	var ac100, accelerators, acpi, acpica, acrn, actions, afs, alpha, amdgfx, amlogic, apparmor, arch, arm, armmsm, armscmi, asahi, aspeed, ath10k, ath11k, ath12k, atm, audit, autofs, axis, b43, batman, bcache, bcachefs, bfs, block, bluetooth, bpf, brcm80211, bridge, broadcom, btrfs, can, ceph, cgroups, chrome, cifs, cirrus, clk, coco, coda, coresight, crypto, csky, cxl, damon, dell, devicetree, dm, dmaengine, drbd, dri, ecryptfs, edac, efi, erofs, etnaviv, exfat, ext4, f2fs, fbdev, fpga, freedreno, fs, fscrypt, fsi, fsverity, fuse, geode, gfs2, gpio, greybus, hams, hardening, hexagon, hfs, hwmon, hyperv, i2c, i3c, ide, iio, imx, input, integrity, intelgfx, intelwiredlan, intelxe, iouring, iomap, iommu, ipack, isdn4linux, isofs, jffs2, jfs, karma, kasan, kbuild, kernel, kernfs, kexec, keyrings, kgdb, kunit, kvm, kvmriscv, kvmx86, kvmarm, leds, libertas, lima, linux1394, linuxppc, linuxpps, livepatching, llvm, loongarch, lsm, lvs, m68k, mapletree, media, mediatek, megaraid, mhi, mips, mjpeg, mm, mmc, modules, mpi3, mptfusion, mptcp, mtd, nbd, net, netdrivers, netfilter, netfs, nfc, nfs, nilfs, nouveau, ntb, ntfs3, nvdimm, nvme, ocfs2, omap, optee, openiscsi, openbmc, openipmi, openrisc, openvpn, openvswitch, openwrt, orangefs, ossdrivers, overlayfs, parisc, parport, pci, perf, phy, pm, ppp, pvrusb2, pwm, qat, raid, rcu, rdma, rds, remoteproc, renesassoc, riscv, rockchip, rpi, rtc, rust, s390, samsungsoc, schedext, scsi, sctp, selinux, serial, sgx, sh, smc, snpsarc, soc, sof, sophgo, sound, spacemit, sparclinux, speakup, spi, spice, squashfs, staging, stm32, sunxi, target, tegra, timestamp, tipc, tls, tomoyo, tqsystems, trace, udf, um, unisoc, usb, usbstorage, v9fs, video, virt, watchdog, wcn36xx, wireguard, wireless, wpan, x25, x86, x86drivers, xen, xfs Subsystem
```

Comment:
Thinking aloud: if every item of this list was on a separate line, we'd have an easier time diffing.

Reply:
This line is not particularly valuable to diff - what is assigned to these variables is much more important.